apps/database/supabase/tests/private-schema-external-chat-observation-replay.sql

```
@@ -0,0 +1,53 @@
+begin;
+select plan(4);
+
+create temporary table observation_context (ws_id uuid primary key, actor_id uuid not null);
+insert into observation_context
+select '77777777-7777-4777-8777-777777777777'::uuid, id
+from public.users order by created_at limit 1;
```

SQLFluff (4.2.2)

[error] The 'order' keyword should always start a new line.

(LT14)